### DISCUSSION TITLE
Error: Could not locate cudnn_ops_infer64_8.dll. Please make sure it is in your library path!

### DISCUSSION THREAD

**Discussion by Yannick013024** [SYSTRAN/faster-whisper#715] [Q&A]
Hi everyone, I tried to get faster-whisper to work, I installed CUDA 11.8 and cuDNN 8.8.1.3. I also installed zlib, as was suggested here: https://github.com/SYSTRAN/faster-whisper/issues/85. This is how my environment variables look like: 
user variables: 
![image](https://github.com/SYSTRAN/faster-whisper/assets/161216868/f10c8b86-18cd-4621-810c-59c818c38942)

system variables: 
![image](https://github.com/SYSTRAN/faster-whisper/assets/161216868/2b58619b-2f97-4c9e-b1e3-688349bde2e4)

I also added zlib to both my system and user variables, and copied the contents of the cuDNN folder into the CUDA folder beforehand (https://medium.com/geekculture/install-cuda-and-cudnn-on-windows-linux-52d1501a8805#3e72). I really don't know what I am doing wrong, any advice?

**Comment by Yannick013024** [comment]
For the record, I'm not using anaconda or anything, just plain python 3.11.8

**Comment by Sukhman4coding** [comment]
I am also having the same issue. I tried using older version of cuDNN but they didnt work too.

**Comment by Purfview** [comment]
As a last resort look at this: https://github.com/Purfview/whisper-standalone-win/releases/tag/libs

**Reply by ZerefCG** [reply]
> As a last resort look at this: https://github.com/Purfview/whisper-standalone-win/releases/tag/libs

Works for me, thanks

**Reply by nguyenmai333** [reply]
nice solution, thanks

**Reply by jameeldark2012** [reply]
I've placed them in the bin folder of CUDA and it's working.

**Reply by Paquette1111** [reply]
tysm

**Reply by glaydersen** [reply]
Thanks a lot! Hundreds of "solutions" but only this one worked for me!

**Reply by DrSiemer** [reply]
For Windows users, "cudnn_ops_infer64_8.dll" can be found in "cuBLAS.and.cuDNN_CUDA12_win_v1.7z" (and **not** in the more recent versions). I placed the files in "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.3\bin" (not overwriting existing files) and now it works for me.

CUDA Version: 12.9
Build cuda_12.3.r12.3/compiler.33281558_0
PyTorch CUDA version: 12.1

**Comment by masonintokyo** [comment]
reinstall pytorch with **--force-reinstall** and **--no-cache** command. like
``` pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu124 --force-reinstall --no-cache```

**Comment by RichardGe** [comment]
my solution was:
download `cuDNN v8.1.1 (Feburary 26th, 2021), for CUDA 11.0,11.1 and 11.2`
from:
https://developer.nvidia.com/rdp/cudnn-archive#a-collapse811-111
and put the DLLs  of this package inside the same folder than 'ctranslate2.dll' 
you can get this folder with the command `pip show ctranslate2`

**Reply by LukasSeratowicz** [reply]
Yep, nothing was working, went into the link you provided, picked newest windows 12.x, downloaded and put DLLs into the same folder as you and it worked! (for one of them it will ask if you want to replace it, i said no btw)

Thank you very much!

**Reply by Pu4** [reply]
Thanks man, you saved my evening. Windows 12, cuda 12.8, nothing was working for me until I copied those dlls. Thanks again!

**Reply by GregTrifan** [reply]
W10 with Cuda 12.6, this suggestion got my project working again!

**Reply by alfred-delacosta** [reply]
> my solution was: download `cuDNN v8.1.1 (Feburary 26th, 2021), for CUDA 11.0,11.1 and 11.2` from: https://developer.nvidia.com/rdp/cudnn-archive#a-collapse811-111 and put the DLLs of this package inside the same folder than 'ctranslate2.dll' you can get this folder with the command `pip show ctranslate2`

I'm on Windows 11 and was able to resolve this issue following the steps from @RichardGe but I had to change it slightly because I was using a Python virtual environment. I had to place the DLL's into my virtual environments folders which was: 

__C:\Users\WindowsUserName\VirtualEnvironmentName\Lib\site-packages\ctranslate2__

The venv name is what you would have created originally. Once I placed the .dll files in that folder, my issues were resolved and WhisperX ran as expected.

**Reply by abdullahrs** [reply]
Works on W11 with Cuda 13.0.

**Comment by SuppliedOrange** [comment]
This worked for me
https://github.com/m-bain/whisperX/issues/899#issuecomment-2549063502

Note that I downgraded my ctranslate2 version to 4.4.0 
`pip install ctranslate2==4.4.0`

**Comment by limheat** [comment]
Make sure your file path has no special characters. I had this problem because the file path included [ and ]. I moved the path and it started working normally.
